src/App.js

```diff
@@ -0,0 +1,32 @@
+const http = require('http');
+var path = require('path');
+var dirlister = require('./GetDirectory');
+
+// var directories = path.dirname('/Users/Refsnes/demo_path.js');
+// console.log(directories);
+
+
+// console.log(dirlister)
+// json = dirlister.JSON
+const server = http.createServer((req,res)=>{
```

Comment:
What about using `express` instead?

Reply:
no problem , i'll try to modify it using express